#############################
### Robot messages
#############################
### estimated pose of robot
float32[] robot0
float32[] robot1

### core obstacles of robots
int32[] obs0_X
int32[] obs0_Y
int32[] obs1_X
int32[] obs1_Y

### inflated obstacles around robots
int32[] obs0_inflated_X
int32[] obs0_inflated_Y
int32[] obs1_inflated_X
int32[] obs1_inflated_Y

### predcited obstacles around robots
int32[] obs0_predicted_X
int32[] obs0_predicted_Y
int32[] obs1_predicted_X
int32[] obs1_predicted_Y


